doc/getting-started.md

```diff
@@ -10,7 +10,7 @@ Ignition will choose where to look for configuration based on the underlying pla
 
 The configuration must be passed to Ignition through the designated data source. Please refer to Ignition [config examples][examples] to learn about writing config files. The provided configuration will be appended to the universal base configuration:
 
-```json ignition
+```json ignition dontvalidate
```

Comment:
Adding anything after `ignition` will prevent that config from being valid (it does a literal string equality check against `json ignition`). Will this break the docs build?

Reply:
It will just render the docs as plain JSON. Is this causing the build to fail?

Reply:
> Is this causing the build to fail?

It's not, since it's not merged. If it will just render it as plain json I don't think it should cause the build to fail. Regardless we don't want to lose the ignition styling because of this.